spark/src/test/scala/org/apache/comet/CometExpressionSuite.scala

```diff
@@ -1053,6 +1053,32 @@ class CometExpressionSuite extends CometTestBase with AdaptiveSparkPlanHelper {
     }
   }
 
+  test("Chr with negative and large value") {
+    Seq(false, true).foreach { dictionary =>
+      withSQLConf(
+        "parquet.enable.dictionary" -> dictionary.toString,
+        CometConf.COMET_CAST_ALLOW_INCOMPATIBLE.key -> "true") {
```

Comment:
I don't think that this test relies on any incompatible cast operations, so we can remove this config.

```suggestion
        "parquet.enable.dictionary" -> dictionary.toString) {
```